osquery/sql/virtual_sqlite_table.cpp

```diff
@@ -38,10 +38,16 @@ const char* getSystemVFS(bool respect_locking) {
 Status genSqliteTableRow(sqlite3_stmt* stmt,
                          TableRows& qd,
                          const fs::path& sqlite_db) {
+  bool user_defined_path_column = false;
   auto r = make_table_row();
   for (int i = 0; i < sqlite3_column_count(stmt); ++i) {
     auto column_name = std::string(sqlite3_column_name(stmt, i));
     auto column_type = sqlite3_column_type(stmt, i);
+
+    if (strcasecmp(column_name.c_str(), "path") == 0) {
```

Comment:
`strcasecmp` may be POSIX only, perhaps we can use `boost:iequals`? Or `std::equal` with a lambda for toLower?

Reply:
Yeah, I think you're right. (And is presumably why the windows build failed). Updated to boost.